hub/core/compression.py

```diff
@@ -303,8 +317,11 @@ def verify_compressed_file(
 
 def get_compression(header=None, path=None):
     if path:
-        if path.lower().endswith(".mp3"):
-            return "mp3"
+        # These formats are recognized by file extension for now
+        file_formats = ["mp3", "flac", "wav"]
+        for fmt in file_formats:
+            if path.lower().endswith("." + fmt):
```

Comment:
use str(path).lower() as path can be a posixpath object at times instead of a string (this was causing kaggle tests to fail earlier)